resources/compilers/Linux/xml2st/PLCGenerator.py

```
@@ -0,0 +1,1782 @@
+                                self.TagName, "action_block", action_infos["id"],
+                                "action", i, "inline")),
+                            ("\n", ())], ())
+                        action_infos["content"] = action_name
+                    self.SFCNetworks["Steps"][step_name]["actions"].append(action_infos)
+
+    def GenerateSFCAction(self, action_name, pou):
+        if action_name not in list(self.SFCNetworks["Actions"].keys()):
+            actionContent = pou.getaction(action_name)
+            if actionContent is not None:
+                previous_tagname = self.TagName
+                self.TagName = ComputePouActionName(self.Name, action_name)
```

Ruff (0.8.2)

`ComputePouActionName` may be undefined, or defined from star imports

(F405)